rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_mavlink_interface.h

```diff
@@ -18,90 +18,196 @@
  * limitations under the License.
  */
 
-#ifndef ROTORS_GAZEBO_PLUGINS_MAVLINK_INTERFACE_H
-#define ROTORS_GAZEBO_PLUGINS_MAVLINK_INTERFACE_H
+#include <iostream>
+#include <math.h>
+#include <deque>
+#include <random>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <sdf/sdf.hh>
+#include <stdio.h>
 
 #include <boost/bind.hpp>
 #include <Eigen/Eigen>
 #include <gazebo/common/common.hh>
 #include <gazebo/common/Plugin.hh>
 #include <gazebo/gazebo.hh>
 #include <gazebo/physics/physics.hh>
-#include <mav_msgs/Actuators.h>
-#include <mavros_msgs/mavlink_convert.h>
-#include <ros/ros.h>
-#include <sensor_msgs/Imu.h>
+#include "gazebo/math/Vector3.hh"
+#include "gazebo/transport/transport.hh"
+#include "gazebo/msgs/msgs.hh"
+#include "common/mavlink.h"     // Either provided by ROS or as CMake argument MAVLINK_HEADER_DIR
+#include "common/mavlink_msg_hil_actuator_controls.h"
 
-#include "rotors_gazebo_plugins/common.h"
+#include "common.h"
+//#include "mavlink/v1.0/common/mavlink.h"
+
+#include "CommandMotorSpeed.pb.h"
```

Comment:
I think `MotorSpeed` was what PX4 used and `CommandMotorSpeed` was what ASL used. I have removed `MotorSpeed` and it's unused inclusion in the MAVLink interface plugin.